reconcile/openshift_serviceaccount_tokens.py

```diff
@@ -0,0 +1,91 @@
+import semver
+import base64
+
+import utils.vault_client as vault_client
+import reconcile.queries as queries
+import reconcile.openshift_base as ob
+
+from utils.openshift_resource import OpenshiftResource as OR
+from utils.defer import defer
+
+
+QONTRACT_INTEGRATION = 'openshift-serviceaccount-tokens'
+QONTRACT_INTEGRATION_VERSION = semver.format_version(0, 1, 0)
+
+
+def construct_sa_token_oc_resource(name, sa_name, sa_token):
+    body = {
+        "apiVersion": "v1",
+        "kind": "Secret",
+        "type": "Opaque",
+        "metadata": {
+            "name": name,
+        },
+        "data": {
+            "token": base64.b64encode(sa_token).decode('utf-8')
+        }
+    }
+    return OR(body, QONTRACT_INTEGRATION, QONTRACT_INTEGRATION_VERSION,
+              error_details=name)
+
+
+def fetch_desired_state(namespaces, ri, oc_map):
+    for namespace_info in namespaces:
+        if not namespace_info.get('openshiftServiceAccountTokens'):
+            continue
+        namespace_name = namespace_info['name']
+        cluster_name = namespace_info['cluster']['name']
+        for sat in namespace_info['openshiftServiceAccountTokens']:
```

Comment:
nit: can we please call this `token` for better readability?

Reply:
Instead of `sat`

Reply:
Nvm, I see that its a rather filled up object and not just the token

Reply:
this is not the `token` per se:
https://gitlab.cee.redhat.com/service/app-interface/merge_requests/3006/diffs#ea2a513d0ff90c2171e9c823a58e7f634be9beaf_81_83